nominations/templates/admin/application_pdf.html

```diff
@@ -99,12 +99,69 @@ <h2>
 
         {{ app.stand_out_information|safe }}
 
+      </section>
+
+      <section>
+
         <h2>
           State of the Race
         </h2>
 
         {{ app.state_of_the_race|safe }}
 
+      </section>
```

Comment:
My understanding is that David is generating these manually right now so a feature flag here isn't required, but can we verify that is the case before we ship PDF changes for something vols won't be using yet.

Reply:
I spoke to David about this, and he said only the admin UI field for volunteers needs to be gated, not pdf changes. Although volunteers have access to the pdf, my impression is that it is mainly just used by staff.